engines/glk/adrift/detection_tables.h

```diff
@@ -695,8 +888,8 @@ const GlkDetectionEntry ADRIFT_GAMES[] = {
 	// ADRIFT One-Hour Game Competition 3
 	DT_ENTRY0("1hgforum2", "5a534ac4e39a319022d145094c46930a", 11185),
 	DT_ENTRY0("1hgcrm", "d97d1ff8f01a61fb477b76df65c77795", 15432),
-	DT_ENTRY1("1hgasdfa", "Competition Release", "fccb2fb890d554263d5f55bc02220ab8", 6440),
-	DT_ENTRY1("1hgdemonhunter", "Competition Release", "ca37aaf35fb15a40a7f5f8caa1475112", 4169),
+	DT_ENTRY1("1hgasdfa", "Comp Rel", "fccb2fb890d554263d5f55bc02220ab8", 6440),
```

Comment:
Not really a fan of this change. "Comp Rel" is not really clear, I'd keep "Competition Release".
Same thing for the other entries, I'd rather use "Release" instead of just "Rel"

Reply:
@tag2015 PR update. Can it be merged now if ok?